src/app/view.js

```diff
@@ -65,25 +65,41 @@ export class View {
         }
     }
 
+
     autoCalculate() {
-        let summerEnergyHT = document.getElementById("summerEnergyHT").value;
-        let summerNetHT = document.getElementById("summerNetHT").value;
-        let summerEnergyNT = document.getElementById("summerEnergyNT").value;
-        let summerNetNT = document.getElementById("summerNetNT").value;
-        let winterEnergyHT = document.getElementById("winterEnergyHT").value;
-        let winterNetHT = document.getElementById("winterNetHT").value;
-        let winterEnergyNT = document.getElementById("winterEnergyNT").value;
-        let winterNetNT = document.getElementById("winterNetNT").value;
-
-        function additionEnergyNet(p1, p2) {
-            let result = parseFloat(p1) + parseFloat(p2);
-            return result
-        }
+        let x = document.getElementsByClassName('consumptionRateInput');
+        var i;
+        for (i = 0; i < x.length; i++) {
```

Comment:
du kannst die var i auch hier definieren => for (var i = 0; i < x.length; i++)
Allgemein ist dies aber eine fehleranfällige for-schlaufe, besser du nutzt for (i in x) {}
dann mit besserer namenswahl x=> consumptionRateInputElements, i => consumptionRateInputElement:

for (var consumptionRateInputElement in consumptionRateInputElements) {
  //code hier
}